### PR TITLE
feat: backer health script

### PR DESCRIPTION
**Pull request by manuelpolzhofer**

### CODE REVIEW

File: bin/backer-health [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+echo "Collateral: $COLLATERAL"
+echo "Lender: $LENDER"
+
+
+message Backer Balances
+echo "Backer Address: $BACKER_ETH_FROM"
+echo "ETH Balance: $(seth --from-wei $(seth balance $BACKER_ETH_FROM)) ETH"
+
+BALANCE=$(seth call $CURRENCY 'balanceOf(address)' $BACKER_ETH_FROM)
+echo "Backer Currency: $(seth --from-wei $(node <<< "console.log(parseInt('$BALANCE', 16))")) DAI"
```
Comment (manuelpolzhofer) [inline bin/backer-health]: @lucasvo  `seth` is not supporting hex to decimal conversion, yet.

For `printf` the number was to big.  I didn't want to spend too much time on it, so I used `node` for know.

Comment (manuelpolzhofer) [reply bin/backer-health]: `echo "ibase=16; HEXNUMBER" | bc` can't handle `0x` prefix and lowercase letters.

Comment (lucasvo) [reply bin/backer-health]: This does the trick: 
```suggestion
echo "Backer Currency: $(seth --from-wei $((16#${BALANCE:2}))) DAI"
```

Comment (manuelpolzhofer) [reply bin/backer-health]: Doesn't work either. We will keep the dependancy for now.

**Pull request merged**